Earthfile

```diff
@@ -0,0 +1,35 @@
+VERSION 0.8
+
+
+INSTALL_PACKAGE: 
+  FUNCTION
+  ARG package
+  COPY +${package}/${package}*.deb ./
+  RUN dpkg -i *.deb
+
+dpkgbuild:
+  FROM debian:bookworm
```

Comment:
I know this is coming from the infra-variants repo, but it may be worth explaining in a comment _why_ debian:bookworm is the base image fro building debs: because it is the oldest debian platform we support. So debs built with it _should_ work on both Bookworm and later Ubuntu & Debian suites.